### PR TITLE
RUN-5489   BrowserView name should be unique

### PR DESCRIPTION
**Pull request by aziz512**
#### Description of Change
Add a check of whether the name provided in options is already in use within the same application by either another browser view or a window.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title starts with the JIRA ticket [pull request process](https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process)
- [x] PR has assigned reviewers

#### Release Notes

Notes: Limited BrowserView identities to unique names

@pbaize @rdepena @bschwinn

### CODE REVIEW

File: src/browser/api/browser_view.ts [hunk 1]
```diff
@@ -19,6 +19,12 @@ export interface BrowserViewOpts extends BrowserViewCreationOptions {
 }
 
 export async function create(options: BrowserViewOpts) {
+    // checking if the name-uuid combination is already in use
+    const { uuid, name } = options;
+    if (getWindowByUuidName(uuid, name) || getBrowserViewByIdentity({ uuid, name })) {
```
Comment (pbaize) [inline src/browser/api/browser_view.ts]: Should we check frames too?

Comment (aziz512) [reply src/browser/api/browser_view.ts]: Done :+1:

File: src/browser/api/browser_view.ts [hunk 2]
```diff
@@ -19,6 +22,12 @@ export interface BrowserViewOpts extends BrowserViewCreationOptions {
 }
 
 export async function create(options: BrowserViewOpts) {
+    // checking if the name-uuid combination is already in use
+    const { uuid, name } = options;
+    if (getWindowByUuidName(uuid, name) || getBrowserViewByIdentity({ uuid, name }) || getInfoByUuidFrame({ uuid, name })) {
+        throw new Error('Provided name-uuid combination is already in use');
```
Comment (pbaize) [inline src/browser/api/browser_view.ts]: Can we make this message more inline with the window one? Let's also add what the uuid/name combination is in the message.

Comment (aziz512) [reply src/browser/api/browser_view.ts]: :+1:

**Pull request merged**